deploy/template.yaml

```diff
@@ -2410,11 +2410,15 @@ Resources:
           AttributeType: "S"
         - AttributeName: "credentialIssuer"
           AttributeType: "S"
+        - AttributeName: "dateCreated"
+          AttributeType: "S"
       KeySchema:
         - AttributeName: "userId"
           KeyType: "HASH"
         - AttributeName: "credentialIssuer"
           KeyType: "RANGE"
+        - AttributeName: "dateCreated"
+          KeyType: "RANGE"
```

Comment:
Are you sure this is right - I think this is updating the primary key of the table, which would still require us to specify the userId when querying (and I'm not sure if updating the primary key is even allowed)?

I was expecting to see a `GlobalSecondaryIndexes` block instead (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html#cfn-dynamodb-table-globalsecondaryindexes)